config/environments/production.rb

```diff
@@ -52,13 +60,16 @@
 
   # Use the lowest log level to ensure availability of diagnostic information
   # when problems arise.
+  # Include generic and useful information about system operation, but avoid logging too much
+  # information to avoid inadvertent exposure of personally identifiable information (PII).
   config.log_level = :info
 
   # Prepend all log lines with the following tags.
   config.log_tags = [:request_id]
 
   # Use a different cache store in production.
   config.cache_store = :dalli_store
```

Comment:
Just a heads up, you may need to swap this out: https://github.com/alphagov/collections/pull/2287/commits/d8a6a51378e8bc37122a558e895ab7610916f179